Project3/index1.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Since there are 14 of these files, I'm only going to comment on this one since the issues are consistent between them:
1. The `head` is in the wrong place; it should be after the `title` on line 8.
1. Inline CSS instead of keeping all your styles in the stylesheet.
1. Overuse of the `br` element. See my notes on your `index.html` for more detail about this.
1. Setting CAPS in your markup instead of CSS.

Your HTML should look something more like this:
```html
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="styles.css">
    <title>10:00 P.M.</title>
</head> <!-- HEAD element goes here -->
<!-- BODY element follows HEAD  -->
<!-- Deleted inline CSS -->

<body>
    <!-- CARD classname doesn't make sense in this context? I'd recommend something like site-container or site-wrapper instead.  -->
    <div class="site-wrapper">
        <h1>10:00 P.M.</h1>
        <h2>Succumb to Fatigue</h2>
        <!-- Use P elements to achieve line breaks or use CSS to control the width of the your copy.  -->
        <p>Days of neglecting good sleep habits finally catches up with me.</p>
        <p>I'm not as young as used to be.</p>
        <p>Gone are the days when I could pull all-nighter after all-nighter.</p>
        <p>What did I even accomplish?</p>
        <p>I lost in Madden, I watched mindless YouTube videos and my DMs are dry...</p>
        <!-- Don't write markup in all caps, handle that via CSS -->
        <p class="all-caps">Nothing, I accomplisehd nothing.</p>
        <!-- Don't use the BR element to achieve what you can do via CSS -->
        <img src="Photo1.jpg" alt="Sleeping">
        <a href="index.html"><span class="all-caps">Back</span> to Timeline</a>
    </div>
</body>

</html>
```